src/schemas/json/project.json

```
@@ -247,6 +247,15 @@
               "description": "Defines glob patterns and file paths to include for copying to build output."
             }
           ]
+        },
+        "xmlDoc": {
+          "type": "boolean"
+        },
```

```suggestion
        "xmlDoc": {
          "description": "",
          "type": "boolean",
          "default": false
        },
```

Warning: [parsing documentation failed, no `xmlDoc` found](https://github.com/dotnet/cli/blob/rel/1.0.0-rc3/Documentation/specs/runtime-configuration-file.md).
Optional fix: add `description` and `default` keys.
Warning: [parsing documentation failed, no `description` and `default` values found, empty string and false used as a fallback](https://github.com/dotnet/cli/blob/rel/1.0.0-rc3/Documentation/specs/runtime-configuration-file.md).